src/ext/resources/legacygraphqlapi/schema.graphqls

```diff
@@ -1376,6 +1376,8 @@ type Leg {
     dropOffBookingInfo: BookingInfo
 
     pickupBookingInfo: BookingInfo
+
+    nextLegs(numberOfLegs: Int, modesWithParentStation: [Mode]): [Leg]
```

Comment:
Add comments for this field and for the `modesWithParentStation` parameter. What does the parameter do? The name isn't exactly self-explanatory.

Reply:
The default can be set here like `numberOfLegs: Int = 2`.

Reply:
What happens if one searches for 5 more legs but only 2 can be found? Does it return 2 legs or 5 where the last 3 are null? I wonder if the `Leg` is actually nullable or not? If it's never null, then the return type could be [Leg!].

Reply:
Probably also makes sense to add a comment for the `numberOfLegs` to explain if always n elements are returned or is it possible that it returns less the requested number.

Reply:
After thinking about this over the weekend, maybe we should not have a default for the `numberOfLegs` at all in OTP as the default value doesn't really come naturally and is just some random number.

Reply:
That would make sense given the apparent randomness of the default but I think `AlternativeLegs.getAlternativeLegs` requires a number as a parameter.  So there would probably effectively be a default value regardless of whether it is stated in the schema or data fetcher

Reply:
It's possible to set the argument as mandatory `numberOfLegs: Int!` and I think the query will then fail if it is not set.

Reply:
Yes, that seems to be the case - made the argument mandatory. There is also some documentation regarding the arguments now